packages/fuzzer/fuzzing_sync.cpp

```diff
@@ -36,9 +40,22 @@ std::optional<FuzzTargetInfo> gFuzzTarget;
 // This is only necessary in the sync fuzzing case, as async can be handled
 // much nicer directly in JavaScript.
 volatile std::sig_atomic_t gSignalStatus;
+std::jmp_buf errorBuffer;
 } // namespace
 
-void sigintHandler(int signum) { gSignalStatus = signum; }
+void sigintHandler(int signum) {
+  gSignalStatus = signum;
+}
+
+// This handles signals that indicate an unrecoverable error (currently only segfaults).
+// Our handling of segfaults is odd because it avoids using our Javascript method to print and instead prints a message
```

Comment:
These lines seem pretty long. Could you please run `npm run format:fix` in the root directory?